kroxylicious-filters/kroxylicious-oauthbearer-validation/src/main/java/io/kroxylicious/proxy/filter/oauthbearer/sasl/ExponentialJitterBackoffStrategy.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter.oauthbearer.sasl;
+
+import java.time.Duration;
+import java.util.Random;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+public class ExponentialJitterBackoffStrategy implements BackoffStrategy {
```

Comment:
I think the duplication is tolerable for now.   We've got a couple of places where this has happened (#1218).  We need to think about where best to put shared code like this.  

I don't think we need to fix this for this code to be merged.